package.json

```diff
@@ -299,8 +299,8 @@
     "@types/sinon-chai": "^3.2.8",
     "@types/uuid": "^8.3.4",
     "@types/vscode": "1.66.0",
-    "@typescript-eslint/eslint-plugin": "^5.22.0",
-    "@typescript-eslint/parser": "^5.22.0",
+    "@typescript-eslint/eslint-plugin": "^6.18.1",
+    "@typescript-eslint/parser": "^6.18.1",
```

Comment:
Otherwise you get this error:

```
Parsing error: DeprecationError: 'originalKeywordKind' has been deprecated since v5.0.0 and can no longer be used. Use 'identifierToKeywordKind(identifier)' instead.
```

https://stackoverflow.com/questions/76996326

Reply:
gotcha thanks!